tests/v2/validation/provisioning/permutations/permutations.go

```diff
@@ -231,7 +232,7 @@ func RunPostClusterCloudProviderChecks(t *testing.T, client *rancher.Client, clu
 
 			services.VerifyAWSLoadBalancer(t, client, lbServiceResp, status.ClusterName)
 		} else if strings.Contains(testClusterConfig.CloudProvider, "external") && providers[0] == provisioninginput.VsphereProviderName.String() {
-			err := charts.InstallVsphereOutOfTreeCharts(client, nodeTemplate, catalog.RancherChartRepo, rke1ClusterObject.ID)
+			err := charts.InstallVsphereOutOfTreeCharts(client, nodeTemplate, catalog.RancherChartRepo, rke1ClusterObject.Name)
```

Comment:
why did this change? I see the jenkins run is passing but still curious.

Reply:
Great question. It's related to the Shepherd part, specifically this: https://github.com/rancher/shepherd/pull/102/files#diff-02b2cdf5fd90d5b47fb90d500fb6a947cb34c0fd10a07a496e2a0f5a56e3def9R33-R36. Charts Install Options now needs ClusterMeta struct, which stores provider type and other various cluster information. That ClusterMeta object is constructed with a cluster name parameter. 

This is a minimal touch on the validation part whereas the Shepherd part mainly handles this construction for this example.